php/shared/usr/local/share/env/55-stack

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-export WEB_DIRECTORY=${WEB_DIRECTORY:-web}
```

Comment:
Why did you move `WEB_DIRECTORY` earlier? It seems kind of unrelated.

Reply:
@TerjeBr it was because Flex moves the web directory to "public", the defaults when it was in 45-framework needed to be before the *-stack, so left 55-stack as the only option. Since 45-framework is now 37-framework and before 40-stack, it was able to be moved back to where it was before